backend/ticketvise/views/api/user.py

```diff
@@ -17,7 +17,7 @@ class Meta:
         fields = ["first_name", "last_name", "email", "username", "avatar_url", "id"]
 
 
-class UserRoleApiView(UserIsInboxStaffMixin, View):
+class UsersRolesApiView(UserIsInboxStaffMixin, View):
```

Comment:
Is deze naam wel handig? Het gaat om een enkele rol toch?

Reply:
Naam veranderd, nu `UserRoleByIdApiView`